qa/common/src/main/java/org/eclipse/kapua/qa/common/utils/InitShiro.java

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua.qa.common.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+
+import org.apache.shiro.SecurityUtils;
+import org.apache.shiro.UnavailableSecurityManagerException;
+import org.apache.shiro.config.Ini;
+import org.apache.shiro.config.IniSecurityManagerFactory;
+import org.apache.shiro.mgt.SecurityManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import cucumber.api.java.en.Given;
+import cucumber.runtime.java.guice.ScenarioScoped;
+
+@ScenarioScoped
+public class InitShiro {
+
+    private static final Logger logger = LoggerFactory.getLogger(InitShiro.class);
+
+    @Given("^Init Security Context$")
+    public void start() throws IOException {
+        try {
+            SecurityManager securityManager = SecurityUtils.getSecurityManager();
+            logger.info("Found Shiro security manager {}", securityManager);
+        }
+        catch (UnavailableSecurityManagerException e) {
+            logger.info("Init shiro security manager...");
```

Comment:
`UnavailableSecurityManagerException` is eaten. 
Add log at least of the message Exception

Reply:
It's the scope of this code. There is no needs to rethrow an exception if you already know what to do when happens.
If the Shiro context it's already configured the test code proceeds, otherwhise, in the catch block the context it's initialized.

Reply:
I said just to log the `Exception` not re-throw it 😁

Reply:
There is NO sense. It's not an Exception, it's like a check.
Instead of writing
if (!security manager defined)
It's used the try catch pattern